includes/class-content-toc.php

```diff
@@ -118,11 +138,12 @@ public function shortcode( $atts, $content = null ) {
 				$item_text = strip_tags( $item_match_arr[1] );
 
 				// Add filter to allow custom TOC item markup
-				$toc_list .= apply_filters(
+				$items_html .= apply_filters(
```

Comment:
This filter does no harm being here but it's probably not going to be used much, you can spend too much time worrying about these things - it's best to make the decision that works for you and then someone will request a filter or feature if it doesn't quite work for them.